### PR TITLE
Add current page to Pagination params

### PR DESCRIPTION
**Pull request by laurakwhit**
## What was changed

This adds the current `page` to query parameters for the `Pagination` component.

## Why?

The page url now has a page number for easy linking and quick access.

## Checklist

1. Closes: **Add current page to Pagination query parameters** 

2. How was this tested:

- [ ] Verify selecting `next` updates the page number in the search params correctly
- [ ] Verify selecting `previous` updates the page number in the search params correctly
- [ ] Verify change the page number in the search params takes the user to the correct page
- [ ] Verify defining a `startingIndex` for a `Pagination` component takes the user to the correct page and the page is listed in the search params
3. Any docs updates needed? n/a

### CODE REVIEW

File: src/lib/holocene/pagination.svelte [hunk 1]
```diff
@@ -37,6 +38,14 @@
     }
   });
 
+  const handlePageChange = () => {
```
Comment (laurakwhit) [inline src/lib/holocene/pagination.svelte:41]: Wondering if we want to also call `handlePageChange` `onMount` or not? It doesn't seem necessary if the first page is always going to be `1`.  However, if a `startingIndex` has been defined for the `Pagination` component the component loads to the corresponding page (which can be something other than `1`) and there is no `page` defined in the search params.

Comment (Alex-Tideman) [reply src/lib/holocene/pagination.svelte:41]: What's the difference between the startingIndex and the page? Just trying to understand what we would using the startingIndex for

Comment (Alex-Tideman) [reply src/lib/holocene/pagination.svelte:41]: Oh I see it's basically the same thing, it's just that you set the starting page with a prop. Yeah I think what you could do is an onMount and if startingIndex is not 0, update the currentPage and call handlePageChange.

Comment (laurakwhit) [reply src/lib/holocene/pagination.svelte:44]: Updated ✅

**Review by Alex-Tideman — APPROVED** [review]

File: src/lib/holocene/pagination.svelte (hunk 1 above)
Comment (Alex-Tideman) [reply src/lib/holocene/pagination.svelte:44]: 👍

**Pull request merged**